Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -84,6 +90,20 @@ static inline D2D_RECT_F __CGRectToD2D_F(CGRect rect) {
 
     // Alpha Blending
     CGFloat alpha = 1.0f;
+
+    inline void ComputeStrokeStyle(ID2D1Factory* factory) {
+        if (strokeStyle) {
+            return;
+        }
+
+        std::vector<CGFloat> adjustedDashes(dashes.size());
+        std::transform(dashes.cbegin(), dashes.cend(), adjustedDashes.begin(), [this](const float& f) -> CGFloat { return f / lineWidth; });
```

Comment:
0 doesn't bother it, but i'll bail out here anyway

---
In reply to: [83978267](https://github.com/Microsoft/WinObjC/pull/1181#discussion_r83978267) [](ancestors = 83978267)